### ISSUE TITLE
[Oracle] Oracle Rpc Request Flooded in the node

### ISSUE THREAD

**Issue by cloud8little** [neo-project/neo-modules#611]
**Summary or problem description**

4 Oracle nodes, and send 3000+ requests continuously, then check the submitoracleresponse requests record. After all 3000+ requests are done ,totally there are 100,000+ submitoracleresponse  requests found (**300+** times of the number of requests ), and the number is still increasing.

Besides, after all requests are on-chain, oracle nodes are still printing logs, sending rpc requests to each other every 3 minutes. 

Oracle config.json as below:

```
{
  "PluginConfiguration": {
    "Network": 5195086,
    "Nodes": ["http://localhost:11332","http://localhost:21332","http://localhost:31332","http://localhost:41332"],
    "MaxTaskTimeout": 432000000,
    "AllowPrivateHost": true,
    "AllowedContentTypes": [ "application/json" ],
    "Https": {
      "Timeout": 5000
    },
    "NeoFS": {
      "EndPoint": "127.0.0.1:8080",
      "Timeout": 15000
    },
    "AutoStart": true
  }
}
```

**Comment by shargon** [comment]
It was not solved after merge #608 ?

**Comment by cloud8little** [comment]
> It was not solved after merge #608 ?

resolved. Close this.
